### DISCUSSION TITLE
Fix Avatar upload policy

### DISCUSSION THREAD

**Discussion by YannHulot** [Etheonor/supanextail#20] [Feedback]
```
create policy "Anyone can upload an avatar."
  on storage.objects for insert
  with check ( bucket_id = 'avatars' );
```

I dont think you should allow anyone to upload files to your bucket. That could leave the door open for unwanted uploads in the bucket. 
Check that the users has an ID before the upload is done.

**Comment by Etheonor** [comment]
Should be fixed with 598fd2671f60bdfb5bce00df5d4a79df7b0cba84

**Reply by YannHulot** [reply]
Nice. Feel free to close this issue.
